sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -126,27 +129,35 @@ async def __aexit__(
         await self.close()
 
     async def open(self):
-        """Opens the connection."""
-        if not self.session and self._session_owner:
-            jar = aiohttp.DummyCookieJar()
-            clientsession_kwargs = {
-                "trust_env": self._use_env_settings,
-                "cookie_jar": jar,
-                "auto_decompress": False,
-            }
-            if self._loop is not None:
-                clientsession_kwargs["loop"] = self._loop
-            self.session = aiohttp.ClientSession(**clientsession_kwargs)
-        # pyright has trouble to understand that self.session is not None, since we raised at worst in the init
-        self.session = cast(aiohttp.ClientSession, self.session)
+        if self._has_been_opened and self._has_been_closed:
```

Comment:
transport.close()
transport.open()
transport.open()   <-- we raise error here? why?

Reply:
As of now we won't, since sessions is not None, double open works fine